hoomd/md/pair/aniso.py

```diff
@@ -443,7 +443,7 @@ class ALJ(AnisotropicPair):
           dimensions. The third dimension in 2D is ignored.
         * ``rounding_radii`` (`tuple` [`float`, `float`, `float`] or `float`,
           **required**) - The semimajor axes of a rounding ellipsoid. If a
-          single value is specified, the rounding ellipsoid is a sphere.
+          single value is specified, the rounding ellipsoid is a sphere. Defaults to 0.0.
```

Comment:
```suggestion
          single value is specified, the rounding ellipsoid is a sphere. 
          Defaults to (0.0, 0.0, 0.0).
```
Wrap to fix flake8 errors and explicitly document the value of the tuple.